apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-expiration/vo-settings-expiration.component.html

```diff
@@ -1,3 +1,4 @@
+<mat-spinner *ngIf="expirationAttribute === undefined" class="ms-auto me-auto"> </mat-spinner>
```

Comment:
the perun loading directive and a loading field which would also work for when saving changes seems more fitting to me